Failed running DateTests on different timezone

Any timezone sensitive test case should accommodate the possibility of running the test on another timezone. I am currently in Chicago, CDT, which is two hours ahead of PDT. Here is the error trace:

```
[ERROR] Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.759 s <<< FAILURE! - in org.hillview.test.dataStructures.DateTests
[ERROR] testTZ(org.hillview.test.dataStructures.DateTests)  Time elapsed: 0.002 s  <<< FAILURE!
java.lang.AssertionError: expected:<2010-12-27T18:50:44Z> but was:<2010-12-27T16:50:44Z>
	at org.hillview.test.dataStructures.DateTests.testTZ(DateTests.java:99)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   DateTests.testTZ:99 expected:<2010-12-27T18:50:44Z> but was:<2010-12-27T16:50:44Z>
```

--- a/platform/src/test/java/org/hillview/test/dataStructures/DateTests.java
+++ b/platform/src/test/java/org/hillview/test/dataStructures/DateTests.java
@@ -90,11 +90,14 @@ public final class DateTests extends BaseTest {
                 .atStartOfDay(ZoneOffset.UTC)
                 .toInstant();
         Assert.assertEquals(expected, instant);
+    }
 
-        parsing = new DateParsing("2010-12-27T10:50:44.000-08:00");
-        instant = parsing.parseDate("2010-12-27T10:50:44.000-08:00");
-        expected = LocalDateTime.of(2010, 12, 27, 10, 50, 44)
-                .atZone(ZoneOffset.systemDefault())
+    @Test
+    public void testTZ1() {
+        DateParsing parsing = new DateParsing("2010-12-27T10:50:44.000-08:00");
+        Instant instant = parsing.parseDate("2010-12-27T10:50:44.000-08:00");
+        Instant expected = LocalDateTime.of(2010, 12, 27, 10, 50, 44)
+                .atZone(ZoneOffset.ofHours(-8))
                 .toInstant();
         Assert.assertEquals(instant, expected);
     }
@@ -110,21 +113,6 @@ public final class DateTests extends BaseTest {
         ldt = parsing.parseLocalDate("2017-10-05T14:05:00.000");
         repr = Converters.toString(ldt);
         Assert.assertEquals("2017/10/05 14:05", repr);
-    }
-
-    //@Test
-    public void offsetTest() {
-        // This test seems to exhibit a bug in the JS interpreter
-        OutputStream stdout = new ByteArrayOutputStream();
-        Context context = Context.newBuilder("js").out(stdout).build();
-        String func = "function f() { print(new Date().getTimezoneOffset()); print(new Date(0).getTimezoneOffset()); }";
-        context.eval("js", func);
-        Value function = context.eval("js", "() => f()");
-        assert function.canExecute();
-        function.execute();
-        String[] debug = stdout.toString().split("\n");
-        Assert.assertEquals(2, debug.length);
-        Assert.assertEquals(debug[0], debug[1]);
     }
 
     @Test

